src/main/java/at/hannibal2/skyhanni/features/inventory/shoppinglist/CategoryTemplate.kt

```diff
@@ -0,0 +1,34 @@
+package at.hannibal2.skyhanni.features.inventory.shoppinglist
+
+import com.google.gson.annotations.Expose
+
+class CategoryTemplate {
```

Comment:
I'm not gonna suggest the change on each of them individually, as there are a _lot_, but do not define explicit `constructor()`s for Kotlin classes where it isn't necessary to.

```suggestion
class CategoryTemplate(
	@Expose val name: String,
	@Expose val color: Char = '6',
	@Expose val hidden: Boolean = false,
	@Expose val items: List<ItemTemplate> = emptyList(),
) {
```

Reply:
why? it just simplifies the code in the respective place it's called, doesn't it?